### PR TITLE
trivial: discover libdrm, and ensure it has amdgpu support using pkgc…

### PR DESCRIPTION
**Pull request by superm1**
…onfig

Fixes: #6266

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/amd-gpu/meson.build [hunk 1]
```diff
@@ -16,6 +16,6 @@ plugin_builtins += static_library('fu_plugin_amd_gpu',
   include_directories: plugin_incdirs,
   link_with: plugin_libs,
   c_args: cargs,
-  dependencies: plugin_deps,
+  dependencies: plugin_deps
```
Comment (hughsie) [inline plugins/amd-gpu/meson.build:19]: total nit, but i think we do want the trailing comma in most cases

File: meson.build [hunk 2]
```diff
@@ -537,7 +537,9 @@ configure_file(
   configuration: conf
 )
 
-amdgpu = cc.has_header_symbol('drm/amdgpu_drm.h', 'AMDGPU_INFO_VBIOS_INFO', required: get_option('plugin_amdgpu'))
+libdrm_amdgpu = dependency('libdrm_amdgpu',
+                           version: '>= 2.4.113',
+                           required: get_option('plugin_amdgpu'))
```
Comment (hughsie) [inline meson.build:542]: out of interest -- how much of amdgpu_drm.h do we actually need? if it's < 20 lines maybe copy and paste might be a good idea?

**Comment by superm1** [comment]
I just realized _it's not libdrm_; it's _the kernel headers_.  This is the wrong change.

File: meson.build (hunk 2 above)
Comment (superm1) [reply meson.build:542]: We need probably about 50 lines from there, but it depends on drm.h which is a lot bigger.

**Pull request closed without merging**